src/applications/gi/components/vet-tec/VetTecContactInformation.jsx

```diff
@@ -3,62 +3,175 @@ import PropTypes from 'prop-types';
+              {institution.physicalAddress2 && (
+                <div>{institution.physicalAddress2}</div>
+              )}
+              {institution.physicalAddress3 && (
+                <div>{institution.physicalAddress3}</div>
+              )}
+              <div>
+                {institution.physicalCity}, {institution.physicalState}{' '}
+                {institution.physicalZip}
+              </div>
             </div>
           </div>
-        </div>
-        <div className="vads-l-col--12 medium-screen:vads-l-col--6">
-          <h3>Mailing address</h3>
-          <div>
-            {institution.address1 && <div>{institution.address1}</div>}
-            {institution.address2 && <div>{institution.address2}</div>}
-            {institution.address3 && <div>{institution.address3}</div>}
+          <div className="vads-l-col--12 medium-screen:vads-l-col--6">
+            <h3>Mailing address</h3>
             <div>
-              {institution.city}, {institution.state} {institution.zip}
+              {institution.address1 && <div>{institution.address1}</div>}
+              {institution.address2 && <div>{institution.address2}</div>}
+              {institution.address3 && <div>{institution.address3}</div>}
+              <div>
+                {institution.city}, {institution.state} {institution.zip}
+              </div>
             </div>
           </div>
         </div>
       </div>
-    </div>
-
-    {/* Production flag for 19534 */}
-    {!environment.isProduction() &&
-      institution.versionedSchoolCertifyingOfficials &&
-      institution.versionedSchoolCertifyingOfficials.length > 0 && (
-        <div>
-          <div className="vads-u-margin-top--4">
-            <h3>School certifying officials</h3>
+    );
+  }
+  return (
+    <div>
+      {institution.physicalAddress1 && (
+        <div className="vads-l-row vads-u-margin-y--4">
+          <div className="vads-l-col--12 medium-screen:vads-l-col--3">
+            <h4 className="contact-heading">Physical address</h4>
           </div>
-          <div className="vads-l-grid-container--full">
-            <div className="vads-l-row">
-              {institution.versionedSchoolCertifyingOfficials.map(sco =>
-                VetTecScoContact(sco),
+          <div className="vads-l-col--9">
+            <div>
+              {institution.physicalAddress1 && (
+                <div>{institution.physicalAddress1}</div>
+              )}
+              {institution.physicalAddress2 && (
+                <div>{institution.physicalAddress2}</div>
               )}
+              {institution.physicalAddress3 && (
+                <div>{institution.physicalAddress3}</div>
+              )}
+              <div>
+                {institution.physicalCity}, {institution.physicalState}{' '}
+                {institution.physicalZip}
+              </div>
             </div>
           </div>
         </div>
       )}
-  </div>
-);
+      {institution.physicalAddress1 && (
```

Comment:
This should be institution.address1, right?

Reply:
That is correct, pushing an updated commit for this soon, once I make Matt's recommended changes.